jobrunner/executors/local.py

```diff
@@ -252,10 +252,23 @@ def finalize_job(job):
         image_id=container_metadata["Image"],
         message=message,
     )
+    output_size_bytes = get_output_size_bytes(job, results.outputs)
+    if output_size_bytes > config.MAX_OUTPUT_SIZE_BYTES:
+        if results.message is None:
+            results.message = f"total size of moderately-sensitive outputs exceeded {config.MAX_OUTPUT_SIZE_BYTES}"
     persist_outputs(job, results.outputs, container_metadata)
     RESULTS[job.id] = results
 
 
+def get_output_size_bytes(job, outputs):
```

Comment:
Possibly miscommunication here - my original suggestion 32mb limit was per file, not across all moderately_sensitive files for an action?

Some actions produce a lot of images (e.g. measures), and this would prevent them from being released.

We may want a perfile limit and also a total limit, perhaps, but the initial 32mb suggestions was a per-file.

I will analyse some file sizes on TPP and see what they currently look like.